sdk/web-pubsub/web-pubsub-express/src/cloudEventsDispatcher.ts

```diff
@@ -136,10 +164,69 @@ function tryGetWebPubSubEvent(req: IncomingMessage): EventType | undefined {
   }
 }
 
+function getStatusCodeFromMqttConnectCode(
+  mqttConnectCode: MqttV311ConnectReturnCode | MqttV500ConnectReasonCode,
+): number {
+  if (mqttConnectCode < 0x80) {
+    switch (mqttConnectCode) {
+      case MqttV311ConnectReturnCode.UnacceptableProtocolVersion:
+      case MqttV311ConnectReturnCode.IdentifierRejected:
+        return 400; // BadRequest
+      case MqttV311ConnectReturnCode.ServerUnavailable:
+        return 503; // ServiceUnavailable
+      case MqttV311ConnectReturnCode.BadUsernameOrPassword:
+      case MqttV311ConnectReturnCode.NotAuthorized:
+        return 401; // Unauthorized
+      default:
+        throw new RangeError(`Invalid MQTT connect return code: ${mqttConnectCode}.`);
```

Comment:
Does throw break the process? maybe not throw but log some error and return 500?